drivers/sensor/bma4xx/Kconfig

```diff
@@ -0,0 +1,31 @@
+# BMA422 3-axis accelerometer config options
+#
+# Copyright (c) 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BMA4XX
+	bool "BMA4XX 3-axis acceleration sensor"
+	default y
+	depends on DT_HAS_BOSCH_BMA4XX_ENABLED
+	depends on SENSOR_ASYNC_API
+	select I2C
+	help
+	  Enable driver for Bosch BMA422 (I2C-based)
+
+config BMA4XX_TEMPERATURE
+	bool "Enable reading the BMA4XX die temperature"
+	default n
+	depends on BMA4XX
+	help
+	  Enable reading the BMA4xx's on-chip temeprature sensor. This creates
+	  extra bus activity and increases code size.
```

Comment:
I'm OK without this if you want to just always support die temperature. Your call.

Reply:
I'll leave it in as long as it doesn't cause trouble. Some of the other drivers support this as an optional feature